moveit_core/robot_state/test/test_kinematic.cpp

```diff
@@ -66,6 +66,20 @@ static bool sameStringIgnoringWS(const std::string& s1, const std::string& s2)
   }
   return i1 == s1.size() && i2 == s2.size();
 }
+static void expect_near_(const Eigen::MatrixXd& x, const Eigen::MatrixXd& y,
+                         double eps = std::numeric_limits<double>::epsilon())
+{
+  ASSERT_EQ(x.rows(), y.rows());
+  ASSERT_EQ(x.cols(), y.cols());
+  for (int r = 0; r < x.rows(); ++r)
+      for (int c = 0; c < x.cols(); ++c)
+          EXPECT_NEAR(x(r,c), y(r,c), eps) << "(r,c) = (" << r << "," << c << ")";
+}
+
+#define expect_near(...) {\
```

Comment:
should be all caps but with perhaps a different post-fix.... EXPECT_NEAR_TRACED?